eclair-core/src/test/scala/fr/acinq/eclair/wire/protocol/PaymentOnionSpec.scala

```diff
@@ -211,7 +211,6 @@ class PaymentOnionSpec extends AnyFunSuite {
       val Right(payload) = FinalPayload.Blinded.validate(decoded, blindedTlvs)
       assert(payload.amount == 561.msat)
       assert(payload.totalAmount == 1105.msat)
-      assert(payload.expiry == CltvExpiry(42))
```

Comment:
Shouldn't that be:

```scala
assert(payload.expiry == CltvExpiry(1234567))
```

Instead of removing the check?

Reply:
Done